packages/ui-extensions-go-cli/create/templates/projects/web_pixel_extension/shopify.ui.extension.toml.tpl

```diff
@@ -1,10 +1,10 @@
 {{ template "shared/shopify.ui.extension.toml" . }}
 runtime_context = "strict"
 
-[configuration]
+[settings]
 type = "object"
```

Comment:
I don't we should be changing this. The object generated is a valid runtime_configuration instance. It follows that standard implementation of other runtime_configurations seen in other extensions. By hiding this value it would complicate the meaning of the settings file.